http-netty/src/main/java/io/micronaut/http/netty/NettyHttpHeaders.java

```diff
@@ -292,8 +295,48 @@ public OptionalLong contentLength() {
         return OptionalLong.empty();
     }
 
+    @Override
+    public List<MediaType> accept() {
+        // use HttpHeaderNames instead of HttpHeaders
+        return MediaType.orderedOf(getAll(HttpHeaderNames.ACCEPT));
+    }
+
+    @Nullable
+    @Override
+    public Charset acceptCharset() {
```

Comment:
is there some performance related motivation for not returning optional here?

Reply:
i saw optional overhead in the profile so i avoided it for new methods. it should be eliminated if the method is inlined but i don't want to rely on it

Reply:
weird because looking at the implementation and where these methods are called they are wrapped in optionals straight a way, or am I missing something?

Reply:
the default implementation uses optional because i just copied the old code. the netty one does not.

the callers still use optional atm. i didn't really try to remove optional, i just avoided it as part of the api for future proofing.